package.json

```diff
@@ -49,6 +49,7 @@
     "gulp-uglify": "^1.4.1",
     "gulp-util": "^2.2.16",
     "gulp-watch": "^0.6.5",
+    "ionic-native": "^1.3.21",
```

Comment:
haha, I was just working on this. It will tie in nicely with what I need. This needs to be in dependencies, not dev-deps, right?